khard/carddav_object.py

```diff
@@ -1078,6 +1082,13 @@ def update(self, input: str) -> None:
         self._set_string_list(self._add_organisation, "Organisation",
                               contact_data)
 
+        # kind
+        self._delete_vcard_object("KIND")
+        try:
+            self.kind = contact_data["Kind"]
+        except KeyError:
+            self.kind = self._default_kind
```

Comment:
The rfc says the field *may* be present so I would not write one if no value is given in the yaml input